bumble/host.py

```diff
@@ -665,18 +684,23 @@ def on_hci_connection_complete_event(self, event):
             )
 
     def on_hci_disconnection_complete_event(self, event):
-        # Find the connection
```

Comment:
Wouldn't it be an error to receive a disconnection event for an unknown handle? Is there a case where we would expect that to happen and would want to still emit an event?